MELA/python/mela_binding.cpp

```diff
@@ -1072,6 +1161,19 @@ PYBIND11_MODULE(Mela, m) {
 
         MAKE_COUPLING_LAMBDA(selfDHwwLambda_qsq, Lambda_w04,  LambdaHIGGS_QSQ_VV_4,  cLambdaHIGGS_VV_QSQ12, 0)
         MAKE_COUPLING_LAMBDA(selfDHwwLambda_qsq, Lambda_w04_h2,  LambdaHIGGS_QSQ_VV_4,  cLambdaHIGGS_VV_QSQ12, 1)
+        
+        /* REUSE COUPLING_C_LAMBDA for Lambda FF N since it is also an integer array */
```

Comment:
This comment should either be removed or changed because we don't use C_LAMBDA here, but rather something new.

Reply:
I removed this in the latest commit